src/shared/featureFlags/openFeature.ts

```diff
@@ -0,0 +1,149 @@
+import { config } from '@grafana/runtime';
+import { OFREPWebProvider } from '@openfeature/ofrep-web-provider';
+import { ClientProviderStatus, OpenFeature, ProviderEvents, type Client, type JsonValue } from '@openfeature/web-sdk';
+
+import { TrackingHook } from './tracking';
+import { getObjectKeys } from '../../shared/utils/utils';
+import { logger } from '../logger/logger';
+
+/**
+ * All feature flags that we intend to use from the GoFF service are defined here.
+ * See {@link https://github.com/grafana/deployment_tools/blob/master/ksonnet/environments/hosted-grafana/waves/feature-toggles/goff/drilldown/metrics/flag-definitions.libsonnet | Metrics Drilldown GoFF flag definitions} for the source of truth.
+ */
+const goffFeatureFlags = {
+  'drilldown.metrics.default_open_sidebar': {
+    valueType: 'string',
+    values: [
+      'treatment', // The sidebar opens by default
+      'control', // The sidebar remains closed (default behavior)
+      'excluded', // The user is excluded from experiment (sidebar remains closed)
```

Comment:
Why `'treatment'` and `'control'` instead of `true`/`false`? We're running an experiment with this feature flag, and the words "Treatment" and "Control" are [experiment design concepts](https://en.wikipedia.org/wiki/Treatment_and_control_groups).

Importantly, [some stacks are excluded](https://github.com/grafana/deployment_tools/blob/9e614b14b9922377f3fe6a60ab9b5ecbf3a3ff49/ksonnet/environments/hosted-grafana/waves/feature-toggles/goff/drilldown/metrics/flag-definitions.libsonnet#L20-L31) from this experiment. So we need more than just `true`/`false`, as it's helpful for statistical purposes to distinguish between (a) users who are not part of the experiment and (b) users for whom the variation is not applied, so that we can compare the treatment group with (b) only.